script/deploy/safe/fixtures/deployment-log.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
what's the purpose of this file?

Reply:
All files in fixtures are used for the unit tests. They act as dummy data.